english/coffee/coffee.md

```diff
@@ -44,3 +44,10 @@ After having your coffee do you want to:
 
 Coffee creates a great medium for people to have conversations about ideas and visions.
 Have chocolate coffee in the morning and [enjoy.](another-coffee/another-coffee.md) 
+
+I always love eating donuts with my coffee. but I don't have any donuts so I drive to the local donut shop to buy some.
+On the way, I get stopped by a police officer.  I pull to the side and the cop knocks on my window and asks for my license
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.